src/app.js

```diff
@@ -27,6 +28,8 @@ import './polyfills';
 
 import configureStore from './configureStore';
 
+setDefaultLocale('nl');
```

Comment:
Shouldn't this line be in the `CalendarInput` component? That's where it is used.
Also, wouldn't `registerLocale('nl', nl);` be enough to use? The `react-datepicker` docs don't mention having to set both, as far as I can see.